research/conditional/utils/argparse.py

```diff
@@ -4,74 +4,113 @@
 def introduce_parser_arguments(
     parser: argparse.ArgumentParser,
 ) -> argparse.ArgumentParser:
-    # core hyperparameters, fixed for all experiments; needs a good reason to change
+    # CORE model hyperparameters, almost always specified in baseline configs
+    parser.add_argument("--model_type", type=str, choices=["gpt", "bert"])
+    parser.add_argument("--ff_mode", type=str, default="vanilla")
```

Comment:
IMO we should go either no defaults or all defaults.

Reply:
not for now

Reply:
trello task added btw